wordpress-mobile/WordPress-iOS
# Remove the .showLoginEmail segue and replace with programmatic nav

Fixes # n/a

See also: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/207

## To test:
1. Check out this branch
2. `rake dependencies`
3. Build and run
4. Navigate to and from each of these flows:
- Login > Continue with WordPress.com - should land on the LoginEmailViewController page.
- Sign up for WordPress.com > Sign up with Email > Enter an email address that has an existing WP.com account and tap Next - it should land on the `LoginEmailViewController` page.

### Do not merge until:
1. The Authenticator PR has been merged: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/207 ✔️
2. A new Authenticator release has been created  ✔️
3. The release is published on Cocoapods trunk ✔️
4. This PR points to the new release ✔️
5. Gridicons 1.0-beta has been released on WPiOS https://github.com/wordpress-mobile/WordPress-iOS/pull/13682

PR submission checklist:
